digitalocean/app/app_spec.go

```diff
@@ -57,6 +58,7 @@ func appSpecSchema(isResource bool) map[string]*schema.Schema {
 		"service": {
 			Type:     schema.TypeList,
 			Optional: true,
+			Computed: true,
```

Comment:
I'm not sure we want this at the top level. I'm curious what the issue was that this fixed?

Reply:
Hmm....I can't remember and the tests now seem to be fine without this change. I'll leave it out for now, I have a created a Jira ticket to investigate the breaking tests in Apps in general. Will leave that out of scope for this ticket.